config/initializers/booleanizer.rb

```diff
@@ -0,0 +1,7 @@
+class String
+  def to_bool
```

Comment:
I often forget that `FOO=1` and `FOO=true` and `FOO=yes` all look like "truth" but aren't.

Similar to this http://www.rubydoc.info/github/gradeawarrior/ruby-development-toolbox/String%3Ato_bool